test_team_new_orleans_fc.py

```diff
@@ -20,7 +20,36 @@ def test_date_time(self):
 
     def test_email_address(self):
```

Comment:
You can remove test_email_address() since you have broken out positive and negative test cases

Reply:
Okay, I will do that.